games-emulation/mgba_libretro/mgba_libretro-0.6.1_20180304.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="A Nintendo Game Boy Advance emulator using the libretro API"
+DESCRIPTION="mGBA is a Nintendo Game Boy Advance emulator with great accuracy. \
+It also now emulates classic Game Boy / Game Boy Color systems. This is the \
+libretro core version of mGBA, for use with RetroArch."
+HOMEPAGE="https://mgba.io"
```

Comment:
Cosmetic suggestion, to have a 100% valid URI ;)
```diff
-HOMEPAGE="https://mgba.io"
+HOMEPAGE="https://mgba.io/"
```
I'll test the recipe in a couple of minutes, but everything already looks OK :)